src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -64,10 +64,9 @@ public EditCommand parse(String args) throws ParseException {
                     .getValue(PREFIX_ADDITIONAL_NOTES).get()));
         }
         if (argMultimap.getValue(PREFIX_CLASS_DATE_TIME).isPresent()) {
-          //TODO after parser is implemented
+            editPersonDescriptor.setClass(ParserUtil.parseClass(argMultimap.getValue(PREFIX_CLASS_DATE_TIME).get()));
```

Comment:
Test case should be added to cover this line. You may refer to `~/tp/src/test/java/seedu/address/logic/commands/EditPersonDescriptorTest.java` for similar test cases for other fields.